tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "outDir": "dist",
     "rootDir": "src",
     "strict": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
I don't think we want this in the rules. Can you have a chat to @lagache about this please?

Reply:
Christophe is ok with it.